src/main/webapp/scripts/poller.js

```diff
@@ -33,10 +33,11 @@
     this.pollingPeriod_ = pollingPeriod;
 
     /**
-     * @private {number} represents the amount of times polling has 
-     *    occured. 
+     * Represents the handler returned by the setTimeout that continues
+     * to poll.
+     * @private {?number}
      */
-    this.attempts_ = 0;
+    this.setTimeoutId_ = null;
```

Comment:
Set it to 0 or -1 or something. Leave null and undefined for non-primitive types with no well-defined default values